### PR TITLE
Revise license test plan.

### PR DESCRIPTION
**Pull request by wadells**
## Description
No major changes, just a couple ease of life tweaks to avoid an
interactive wizard during cert generation and suppress/preempt errors.

## Type of change

* Docs

## Linked tickets and other PRs
Developed & tested while executing #1828

## TODOs
- [x] Self-review the change
- [x] Perform manual testing
- [x] Address review feedback

## Testing done
Not much to be done.  Here is proof the openssl command works without prompting:

```
$ openssl req -newkey rsa:2048 -nodes -keyout domain.key -x509 -days 365 -out domain.crt -subj "/C=US/ST=The Internet/L=127.0.0.1/O=Gravitational/OU=Hastily Generated Values Division/CN=Test"   
Generating a 2048 bit RSA private key                                                          
................................................+++++
........................................................................+++++
writing new private key to 'domain.key'                                                        
----- 
```

Everything else is self explanatory and has output captured in #1852 or #1853.

### CODE REVIEW

File: docs/7.x/testplan.md [hunk 1]
```diff
@@ -311,29 +311,44 @@ license:
...
 
+- [ ] Preempt an expected failure before beginning the next install [#1853](https://github.com/gravitational/gravity/issues/1853)
+```bash
+$ sudo ./gravity leave --force
+```
+
 - [ ] Verify can install in CLI mode with license:
 ```bash
-$ sudo ./gravity install --license="$(cat /tmp/license)"
+$ sudo ./gravity install --license-file=licence.pem"
```
Comment (r0mant) [inline docs/7.x/testplan.md:351]: licen**s**e.pem

**Review by r0mant — APPROVED** [review]

File: docs/7.x/testplan.md [hunk 2]
```diff
@@ -311,29 +311,44 @@ license:
...
 - [ ] Build an encrypted cluster image:
 ```bash
 $ tele build app.yaml --ca-cert=domain.crt --encryption-key=qwe123
 ```
 
 - [ ] Verify can install in wizard UI mode.
-  - [ ] Verify license prompt appears in the UI.
-  - [ ] Insert the generated license and verify the installation succeeds.
-  - [ ] Verify license can be updated via cluster UI after installation.
+ - [ ] Verify license prompt appears in the UI.
```
Comment (bernardjkim) [inline docs/7.x/testplan.md:327]: nit: Add additional space if this was intended to be indented.

**Review by a-palchikov — APPROVED** [review]
lgtm with other comments out of the way

File: docs/7.x/testplan.md (hunk 2 above)
Comment (wadells) [reply docs/7.x/testplan.md]: Fixed in 5770c9e.

File: docs/7.x/testplan.md (hunk 1 above)
Comment (wadells) [reply docs/7.x/testplan.md]: Fixed in 5770c9e.

File: docs/7.x/testplan.md [hunk 3]
```diff
@@ -323,17 +325,30 @@ $ tele build app.yaml --ca-cert=domain.crt --encryption-key=qwe123
...
 
+- [ ] Preempt an expected failure before beginning the next install [#1853](https://github.com/gravitational/gravity/issues/1853)
+```bash
+$ sudo ./gravity leave --force
+```
+
 - [ ] Verify can install in CLI mode with license:
 ```bash
-$ sudo ./gravity install --license="$(cat /tmp/license)"
+$ sudo ./gravity install --license-file=licesse.pem"
```
Comment (knisbet) [inline docs/7.x/testplan.md:351]: This seems like a typo: `licesse.pem`

Comment (wadells) [reply docs/7.x/testplan.md]: Thanks! fixed in b693531.

**Pull request merged**